lib/helpers.js

```diff
@@ -30,7 +30,8 @@ unslugify = function (name) {
  * unrecognized.
  */
 
-var yesPrimary = ['yes', 'yeah', 'y', 'ya', 'yea', 'yup', 'yep', 'true', 'sure', 'gladly', 'absolutely', 'totally', 'totes', 'got it', 'ok', 'k', 'kk', 'umkay', 'sounds good', 'indeed', 'no problem', 'not a problem', 'okay', 'yess', 'yesss', 'yas', 'good', 'so far, so good', 'so far so good', 'of course']
+const neither = ['not sure'] // This is interpreted as 'yes' without a special case
+var yesPrimary = ['yes', 'yeah', 'y', 'ya', 'yea', 'yup', 'yep', 'true', 'sure', 'gladly', 'totally', 'totes', 'got it', 'ok', 'k', 'kk', 'umkay', 'sounds good', 'indeed', 'no problem', 'not a problem', 'okay', 'yess', 'yesss', 'yas', 'good', 'so far, so good', 'so far so good', 'of course']
```

Comment:
Not really a problem you introduced but itd probably be nice to change these all two const, just looks funny in the file.